src/components/Proposals/ProposalBanner.js

```diff
@@ -39,43 +42,62 @@ function ProposalBanner({ onCloseBanner, ...props }) {
         >
           <IconCross
             css={`
-              color: ${theme.surfaceOpened};
+              color: ${theme.surface};
             `}
           />
         </ButtonIcon>
-
-        <h1
-          css={`
-            background: linear-gradient(to bottom, ${theme.accentStart} -50%,${
-            theme.accentEnd
-          } 60%);
-            -webkit-background-clip: text;
-            -webkit-text-fill-color: transparent;
-
-            ${textStyle('title4')}
-            font-size: ${6.5 * GU}px;
-            line-height: 1.1;
-            font-weight: 700;
-            margin-bottom: ${1.5 * GU}px;
-          `}
-        >
-          This is a test
-        </h1>
         <div
           css={`
-            ${textStyle('body1')};
-            max-width: ${95 * GU}px;
-            margin: auto;
-            margin-bottom: ${3.5 * GU}px;
-            color: ${theme.contentSecondary};
+            display: flex;
+            flex-direction: ${compactMode ? `column` : `row`};
+            align-items: center;
+            justify-content: space-around;
           `}
         >
-          Aragon Network Cash is a test organization set up as part of a new
-          round of precedence campaign disputes. Learn more on the{' '}
-          <Link href="https://aragon.org/blog/juror-dispute-guide-aragon-network-cash">
-            Aragon project blog
-          </Link>
-          .
+          <div>
+            <h1
+              css={`
+                color: ${theme.surface};
+                ${textStyle('title4')}
+                font-size: ${compactMode ? 6.5 * GU : 7.75 * GU}px;
+                line-height: 1.1;
+                font-weight: 700;
+                text-align: left;
+                margin-bottom: ${2.5 * GU}px;
+                max-width: ${62 * GU}px;
+              `}
+            >
+              This is a test organization
+            </h1>
+            <div
+              css={`
+                ${textStyle('title4')};
+                font-size: ${compactMode ? 2.5 * GU : 2.75 * GU}px;
+                max-width: ${62 * GU}px;
+                text-align: left;
+                color: ${theme.surface};
+              `}
+            >
+              Aragon Network Cash is a test organization set up as part of a new
+              round of precedence campaign disputes. Learn more on the{' '}
+              <Link
+                href="https://aragon.org/blog/juror-dispute-guide-aragon-network-cash"
+                css={`
+                  color: ${theme.surface};
+                `}
+              >
+                Aragon project blog
+              </Link>
+              .
+            </div>
+          </div>
+
+          <img
+            src={pcBanner}
+            css={`
+              max-width: 90%;
+            `}
+          />
```

Comment:
I'm wondering whether we should only render this at larger sizes, or make it smaller on small screens. It takes quite a bit of space on mobile, what do you think?

![image](https://user-images.githubusercontent.com/11708259/91873243-b1b2d180-ec70-11ea-9715-13ebf9f6b8a1.png)

Reply:
You're right, it was odd. Now it looks like this
![image](https://user-images.githubusercontent.com/10419340/91887243-e880eb80-ec60-11ea-81da-840cce03a08b.png)